pricing-page-exercise/dilara/index.html

```diff
@@ -8,6 +8,101 @@
   <title>Pricing page Exercise</title>
 </head>
 <body>
-  
+  <div class="main-section">
+  <div class="nav-bar">
+    <div class="company-name">
+      <p>Company Name</p>
+    </div>
+    <div class="pages">
+      <a class="link" href="https://getbootstrap.com/docs/4.0/examples/pricing/#">Features</a>
+      <a class="link" href="https://getbootstrap.com/docs/4.0/examples/pricing/#">Enterprise</a>
+      <a class="link" href="https://getbootstrap.com/docs/4.0/examples/pricing/#">Support</a>
+      <a class="link" href="https://getbootstrap.com/docs/4.0/examples/pricing/#">Pricing</a>
+      <button class="sign-up">Sign up</button>
+    </div>
+  </div>
+  <div class="main-page">
+  <div class="header">
+    <h1>Pricing</h1>
+    <p class="information">Quickly build an effective pricing table for your potential customers with this Bootstrap example. It's built with default Bootstrap components and utilities with little customization</p>
+  </div>
+  <div class="options">
+    <div class="free">
+      <div class="header-free">
+        <span>Free</span>
+      </div>
+      <div class="description">
+        <br>
+        <span><b class="currency">$0</b> <span class="grey">/ mo</span></span>
+        <p>10 users included</p>
+        <p>2 GB of storage</p>
+        <p>Email support</p>
+        <p>Help center access</p>
+        <button class="sign-free">Sign up for free</button>
+      </div>
+    </div>
+    <div class="pro">
+      <div class="header-pro">
+        <span>Pro</span>
+      </div>
+      <div class="description">
+        <br>
+        <span><b class="currency">$15</b> <span class="grey">/ mo</span></span>
+        <p>20 users included</p>
+        <p>10 GB of storage</p>
+        <p>Priority email support</p>
+        <p>Help center access</p>
+        <button class="second-button">Get started</button>
+      </div>
+    </div>
+    <div class="enterprise">
+      <div class="header-enterprise">
+        <span>Enterprise</span>
+      </div>
+      <div class="description">
+        <br>
+        <span><b class="currency">$29</b> <span class="grey">/ mo</span></span>
+        <p>30 users included</p>
+        <p>15 GB of storage</p>
+        <p>Phone and email support</p>
+        <p>Help center access</p>
+        <button class="second-button">Contact us</button>
+      </div>
+    </div>
+  </div>
+  <hr>
+  <div class="footer">
+    <div class="icon">
+      <img src="https://getbootstrap.com/docs/4.0/assets/brand/bootstrap-solid.svg" alt="" width="24" height="24">
+      <!--Add Icon Here-->
+      <p class="iconp">© 2017-2018</p>
```

Comment:
I couldn't change what you recommended to me. I left a paragraph because it shouldn't be the same line. Please correct me in this comment so I can see how I should do it.

Reply:
@dssanli all you need to do is to change html like this:

```
       <span class="iconp">© 2017-2018</span>
```

and add this line to CSS file to the .iconp class
```
display: block;
```

Paragraph is a block level element but span is an inline element by default. So when you switch to using <span> from using a <p>, you often need to set a display property to "block". :)